profiling/src/internal/profile.rs

```diff
@@ -162,6 +164,7 @@ impl Profile {
         profile.endpoints.local_root_span_id_label = profile.intern("local root span id");
         profile.endpoints.endpoint_label = profile.intern("trace endpoint");
         profile.timestamp_key = profile.intern("end_timestamp_ns");
+        profile.label_set_start_key = profile.intern("dd_following_labels_belong_to_set");
```

Comment:
Minor: Up until now, most of our labels were separated by spaces, not underscores. The timeline timestamp label broke with this pattern, but I would suggest we don't keep making it more inconsistent from now on lol.